src/main/java/org/folio/inventory/resources/EventHandlers.java

```diff
@@ -46,12 +47,16 @@
 
   private static final String DATA_IMPORT_EVENT_HANDLER_PATH = "/inventory/handlers/data-import";
   private static final String INSTANCES_EVENT_HANDLER_PATH = "/inventory/handlers/instances";
+  // value in ms
+  private static final int DEFAULT_HTTP_TIMEOUT = 3000;
```

Comment:
Rather than use a comment, the unit could be explicit in the name

```suggestion
  private static final int DEFAULT_HTTP_TIMEOUT_IN_MILLISECONDS = 3000;
```